Archives/VolFile.h

```diff
@@ -53,11 +53,20 @@ namespace Archives
 			CompressionType compressionType;
 		};
 
+		// Specify boundary padding for a volume file section
+		enum class VolPadding : uint32_t
+		{
+			TwoByte = 0,
+			FourByte = 1 // Default value
```

Comment:
Rather than a comment, perhaps this can be written as:
```
enum class VolPadding : uint32_t
{
	TwoByte = 0,
	FourByte = 1,
	Default = FourByte
}
```

Reply:
I didn't actually know you could use default like this on a c++ enum. I think that it might be a good idea to put the default in so it is clear to maybe a less astute reader who maybe doesn't notice/understand the optional argument syntax in the constructor.

Reply:
If the constructor then sets the default value using `Default`, it makes the constructor less self-documenting.

If another class is written, where someone wanted to default to 2-byte padding, it would be weird to reuse the existing enum.